lib/restaurant.rb

```diff
@@ -0,0 +1,14 @@
+class Restaurant
+
+  DEFAULT_MENU = {"dumplings" => 3.99, "cheeseburger" => 5.99, "peking duck" => 10.99}
```

Comment:
Metrics/LineLength: Line is too long. [86/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)